### PR TITLE
Update AdvancedConfiguration.stories.mdx

### PR DESCRIPTION
**Pull request by PauwelsYoran**
added IdPrefixProvider to documentation for problems that can be caused by fluentprovider interop

## Related Issue(s)

[[](https://github.com/microsoft/fluentui/issues/27976)](https://github.com/microsoft/fluentui/issues/27976)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/8f385591d246aa1f640e65a9e61149dd49512f5d'>8f385591d246aa1f640e65a9e61149dd49512f5d</a></sub>
<div id="bundleSizeComment9423"/>

File: apps/public-docsite-v9/src/Concepts/AdvancedConfiguration.stories.mdx [hunk 1]
```diff
@@ -110,6 +110,20 @@ function MyComponent(props) {
...
+ Having multiple applications using `FluentProvider` on a single web page can cause interop problems. This will result in styling to be lost on one or multiple components that are using fluent ui.
+ The problem can be solved by adding the `IdPrefixProvider` component around the `FluentProvider`.
+
+```jsx
+<IdPrefixProvider value="APP1-">
+  <FluentProvider>
+    {/* app provider */}
+  </FluentProvider>
+</IdPrefixProvider>
+```
```
Comment (layershifter) [inline apps/public-docsite-v9/src/Concepts/AdvancedConfiguration.stories.mdx:125]:
````suggestion
Having multiple applications using `FluentProvider` on a single web page can cause interop problems when they come from different bundles, more details in [microsoft/fluentui#26496](https://github.com/microsoft/fluentui/pull/26496).

Adding the `IdPrefixProvider` component around the `FluentProvider` will resolve the issue of losing styling on components and IDs collisions in an application.

```jsx
import { IdPrefixProvider, FluentProvider } from '@fluentui/react-components';
import * as React from 'react';

function MyApp() {
  return (
    <IdPrefixProvider value="APP1-">
      <FluentProvider>{/* children */}</FluentProvider>
    </IdPrefixProvider>
  );
}
```
````

**Comment by PauwelsYoran** [comment]
@microsoft-github-policy-service agree

**Review by layershifter — APPROVED** [review]

**Review by ling1726 — APPROVED** [review]

**Comment by 1hw7** [comment]
Hi @layershifter, I'm wondering if it might be a good idea to move this to the introduction section of the docs? Or somewhere more likely for all users to see? We spent quite a while debugging this on my team. We use fluent v9, while a npm package we consume also uses fluent v9 and neither of us were aware of this and the issue was quite hard to track down.

I wonder if this was moved to the introduction section, if others might be less likely to spend as long on this! On that note though, so grateful docs were added on this otherwise we may have still have been trying to figure it out :D

**Comment by layershifter** [comment]
@1hw7 while this issue appears, it does not like a goto option for all audience 🤔

FYI: #28670 added new behavior, newer versions of Fluent will notify you if the problem exists and link to this page.

**Pull request merged**